examples/simple_polaris_client.cc

```diff
@@ -30,6 +30,16 @@ DEFINE_string(polaris_api_key, "",
 DEFINE_string(polaris_unique_id, "",
               "The unique ID to assign to this Polaris connection.");
 
+DEFINE_string(
+    polaris_hostname, "",
+    "Specify an alternate hostname to use when connecting to the Polaris "
+    "corrections network. If blank, use the default hostname.");
```

Comment:
hint that is useful for regional endpoints in the EU and APAC.

Reply:
@anathan Is this clear, or would you suggest anything else?
```
DEFINE_string(
    polaris_hostname, "",
    "Specify an alternate hostname to use when connecting to the Polaris "
    "corrections network. This may be useful when connecting to a regional "
    "endpoint in the EU or APAC. If blank, use the default hostname.");
```